configure

```diff
@@ -330,6 +330,8 @@ for raw_cmake_arg in args.raw_cmake_args:
 
 invocation.append('--')
 invocation.append(project_root)
+invocation.append("-G")
+invocation.append("MSYS Makefiles")
```

Comment:
How should we say to CMake that it should use this generator, but just on WIN32 platform?